### ISSUE TITLE
TypeError: Cannot read property 'simpleText' of undefined

### ISSUE THREAD

**Issue by yatki** [yamalight/youtube-control#15]
Hey Tim,

Getting the error message below:

```
Uncaught (in promise) TypeError: Cannot read property 'simpleText' of undefined
    at bundle.js:6
    at Array.map (<anonymous>)
    at bundle.js:6
    at async _ (bundle.js:6)
    at async init (bundle.js:6)
```

---

When I build code without minifying it points to:

```js
return {
      name: it.formattedTitle.simpleText,
      url: `https://www.youtube.com${it.navigationEndpoint.commandMetadata.webCommandMetadata.url}`,
      thumbnail: (_it$thumbnail = it.thumbnail) === null || _it$thumbnail === void 0 ? void 0 : (_it$thumbnail$thumbna = _it$thumbnail.thumbnails) === null || _it$thumbnail$thumbna === void 0 ? void 0 : (_it$thumbnail$thumbna2 = _it$thumbnail$thumbna[0]) === null || _it$thumbnail$thumbna2 === void 0 ? void 0 : _it$thumbnail$thumbna2.url // original: it,

    };
```

Please let me know if you need more info?

**Comment by yamalight** [comment]
Interesting, a channel without a simple name 🤔 
Could you please try to `console.log` it and share the JSON for breaking channel here?

**Comment by yamalight** [comment]
Should be fixed in [v1.0-pre6](https://github.com/yamalight/youtube-control/releases/tag/v1.0-pre6). Feel free to re-open if that's not the case :)
